utils/deploy.ts

```diff
@@ -125,7 +125,7 @@ export const deployRemovalContract = async ({
   return hre.deployOrUpgradeProxy<Removal, Removal__factory>({
     contractName: 'Removal',
     args: [],
-    options: { initializer: 'initialize()' },
+    options: { initializer: 'initialize()', unsafeAllow: ['delegatecall'] },
```

Comment:
Obligatory why do we need to allow something unsafe?

Reply:
https://app.slack.com/client/T7SRSSUN7/C023A5VN86R/1656529393.031139